mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -645,15 +779,15 @@ ::mediapipe::Status GlAnimationOverlayCalculator::Process(
       texture_ = helper_.CreateSourceTexture(input_texture);
     }
 
+    MP_RETURN_IF_ERROR(GlBind(current_frame, texture_));
     if (has_model_matrix_stream_) {
-      // Bind texture being used
-      MP_RETURN_IF_ERROR(GlBind(current_frame, texture_));
+      // Draw objects using our latest model matrix stream packet.
       for (const ModelMatrix &model_matrix : current_model_matrices_) {
-          MP_RETURN_IF_ERROR(GlRender(current_frame, model_matrix.get()));
+        MP_RETURN_IF_ERROR(GlRender(current_frame, model_matrix.get()));
```

Comment:
If we're using the same rendering function for both, we should make sure that:
(1) The 0.2 alpha we check against in our fragment shader is low enough that we're not discarding pixels accidentally in our Objectron demos, if any of those textures rely on alpha transparency.
(2) This means Objectron rendering will be getting our normal approximations and lighting code.  We should double-check that this doesn't adversely affect how those render (like giving them any "popping" artifacts.  Or else we need to provide a toggle to enable/disable lighting.
(3) Also, this means that we're using our approximate normals and lighting even for invisible "blocker" object rendering.  Rendering will be just fine, but this might mean we're spending a lot more unnecessary GPU work to get the same result there.  Not sure if we care about this right not, but if we don't provide a toggle to turn off lighting/normals, possibly add a TODO for fixing that in the future?